### PR TITLE
Youtube reviews carousel created

### PR DESCRIPTION
**Pull request by sarthaksri-71449**
Frontend:
In every film's detail page. Now there is a carousel which shows different Youtube review videos of that particular film.
Backend:
1. Connected with the database.
2. Reviews schema defined.
3. Routes for getting a movie's review and adding new reviews defined.

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 
 
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-
-  
-  <React.StrictMode>
+  <Auth0Provider
+  domain="dev-iv6sdwq2w07uuaxb.us.auth0.com"
```
Comment (PatelNeelMahesh) [inline client/src/index.js:12]: Keep `clientId` and `domain` in `.env` file.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
Make the required changes before merging the pull request.

File: client/src/index.js (hunk 1 above)
Comment (sarthaksri-71449) [reply client/src/index.js:12]: The requested changes will be there in the next to next commit which is on the way.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

**Pull request merged**